src/content/docs/en/guides/deploy/netlify.mdx

```diff
@@ -45,18 +45,25 @@ If you prefer to install the adapter manually instead, complete the following tw
       adapter: netlify(),
     });
     ```
- 
-    To render your project using [Netlify's experimental Edge Functions](https://docs.netlify.com/netlify-labs/experimental-features/edge-functions/#app) instead, change the `netlify/functions` import in the Astro config file to use `netlify/edge-functions`.
-      ```js title="astro.config.mjs" ins={3} del={2}
-      import { defineConfig } from 'astro/config';
-      import netlify from '@astrojs/netlify/functions';
-      import netlify from '@astrojs/netlify/edge-functions';
-
-      export default defineConfig({
-        output: 'server',
-        adapter: netlify(),
-      });
-      ```
+
+    :::note
+    In version `3.0.0` of `@astrojs/netlify`, the `@astrojs/netlify/functions` package has been enhanced to support Edge middleware directly, eliminating the need for a separate adapter for deploying your entire app to the edge. It's recommended to update your Astro configuration to leverage this enhancement.
+    :::
+
+    Below is the updated configuration snippet. In your `astro.config.mjs`, replace the import statement and update the `adapter` option as shown:
+
+    ```js title="astro.config.mjs" ins={3, 8} del={2}
+    import { defineConfig } from 'astro/config';
+    import netlify from '@astrojs/netlify/edge';
+    import netlify from '@astrojs/netlify/functions';
+
+    export default defineConfig({
+      output: 'server',
+      adapter: netlify({
+        edgeMiddleware: true
+      }),
+    });
+
```

Comment:
This codeblock is never closed, we also are using a new syntax for codeblocks

``````suggestion
    ```diff lang="js"
      // astro.config.mjs
      import { defineConfig } from 'astro/config';
    - import netlify from '@astrojs/netlify/edge';
    + import netlify from '@astrojs/netlify/functions';

      export default defineConfig({
        output: 'server',
        adapter: netlify({
        + edgeMiddleware: true
        }),
      });
    ```
``````